Majasdarbs/03-majasdarbs/Portfolio/kristersgutmanis.html

```diff
@@ -0,0 +1,166 @@
+        <!-- EDUCATION -->
+         <!-- FAQ -->
+         <section>
+             <div class="faq-container">
+            <h2 style="color: #3A5A6A;"> &lt; FAQ &gt;</h2>
+            <details>
+                <summary>My hobbyies?</summary>
+                <ul>
+                    <li>Brazilian jiu jitsu</li>
+                    <li>Art</li>
+                </ul>
+              </details>
+              <details>
+              <summary>Languages</summary>
+                <ul>
+                    <li>Latvian (native)</li>    
+                    <li>English (working Professional)</li>  
+                </ul>
+            </details>
+              <details>
+                <summary>My favorite playlist</summary>
+                <br>
+                <iframe 
+                    src="https://open.spotify.com/embed/playlist/16gXsmg48MdRTqCLNnx9Cu?utm_source=generator" 
+                    width="100%" 
+                    height="80px" 
+                    frameBorder="0" 
+                    allowfullscreen="" 
+                    allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture">
+                </iframe>
+              </details>
+              <h2 style="color: #3A5A6A;"> &lt;/FAQ &gt;</h2>
+            </div>
+        </section>
+        
+    </div>
+ 
+        <!-- SKILLS -->
+        <div class="container-resume">
+        <section>
+            <h2 style="color: #3A5A6A;"> &lt; Skills &gt;</h2>
+            <ul>
+                <li>HTML5</li>
+                <li>CSS3</li>
+                <li>SCSS</li>
+                <li>JavaScript</li>
+                <li>React JS</li>
+                <li>Git</li>
+                <li>Bootstrap</li>
+                <li>Tailwind CSS</li>
+                <li>Adobe Photoshop</li>
+            </ul>
+            <h2 style="color: #3A5A6A;"> &lt;/Skills &gt;</h2>
+        </section>
+    </div>
+        <!-- SKIllS -->
+    </div>
+       
+        <!-- CONTACT FORM -->
+        <section id="contact-me">
+            <div class="flex-container">
+            <h2>Contact me </h2>
+
+            <form action="action_page.php">
+                <input class="input-text" type="text" id="firstname" name="firstname" placeholder="Your name..">
+             
+
+                <input class="input-text" type="text" id="company" name="lastname" placeholder="Company name..">
+            
+            
+                <textarea id="subject" name="subject" placeholder="Write something.."></textarea>
+                <div class="button-container">
+                <button onclick="alert('Forma nosūtīta')" class="contact-form-button" type="submit" >Submit </button>   
+            </div>        
+              </form>
+            </div>
+        </section>
+        <!-- CONTACT FORM -->
+    </main>
+
+    <!-- Code for changing navbar color -->
+    <script>const navbar = document.querySelector('.nav-fixed')
+
+        window.onscroll = function() {
```

Comment:
Onscroll works, but it is performance inefficient. Check this article when you have free time, this is kinda cool implementation that solves the same problem https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API